.yamllint_defaults.yml

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
```

Comment:
It seems to me that this is missing a change for the .yamllint file to extend the .yamllint_defaults.yml file.

Reply:
Good catch.  The auto-maintenance sync-template.sh script was changed to only copy `.yamllint.yml` if it didn't exist, since that is now the file used for local customizations.  Since network already had a .yamllint.yml, I had to manually copy it from template.